ACHNBrowserUI/ACHNBrowserUI/views/turnips/charts/TurnipsChartGrid.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  TurnipsChartGrid.swift
+//  ACHNBrowserUI
+//
+//  Created by Renaud JENNY on 03/05/2020.
+//  Copyright © 2020 Thomas Ricouard. All rights reserved.
+//
+
+import SwiftUI
+import Backend
+
+struct TurnipsChartGrid: Shape {
```

Comment:
bye bye grid? 😂

Reply:
Let's try by removing only the horizontal grid and keeping only the vertical one :)

Reply:
Done